src/databricks_ai_bridge/test_utils/vector_search.py

```diff
@@ -135,13 +135,13 @@ def _get_index(
 
 @pytest.fixture(autouse=True)
 def mock_workspace_client() -> Generator:
-    def _get_table_comment(full_name: str) -> MagicMock:
-        table = MagicMock()
-        table.comment = "Mocked table comment"
-        return table
+    def _get_serving_endpoint(full_name: str) -> MagicMock:
```

Comment:
QQ why is this mock needed?

Reply:
This mock is still needed when fetching the resources for vectorSearch. We check if the embedding endpoint is databricks